R/length.R

```diff
@@ -160,7 +152,17 @@ xportr_length <- function(.df,
       select(any_of(c(variable_name, "length_df", "length_meta")))
 
     max_length_msg(length_msg, verbose)
+
+    # Build `width` attribute
+    length_data[names(.df)]
+  }
+
+  for (i in names(.df)) {
+    attr(.df[[i]], "width") <- width_attr[[i]]
   }
 
+  # Message for missing var and missing length
+  length_log(miss_vars, miss_length, verbose)
```

Comment:
This can safely be moved as "else" condition doesn't change any of these variables)